openpdf/src/test/java/com/lowagie/text/pdf/SimplePdfTest.java

```diff
@@ -60,4 +75,60 @@ void testTryWithResources_with_unknown_os() throws Exception {
             cb.addTemplate(page, 1, 0, 0, 1, 0, 0);
         }
     }
+    
+    @Test
+    void testCrossReferenceTableEncoding() throws Exception {
```

Comment:
It would make sense to move this added test in a new class (e.g. `CrossReferenceTableEncodingTest`).

Reply:
Done!